interfaces/builtin/bluez.go

```diff
@@ -138,6 +138,9 @@ dbus (send)
     path=/org/freedesktop/hostname1
     interface=org.freedesktop.DBus.Introspectable
     member=Introspect,
+
+# Access to device buttons
+/dev/uinput rw,
```

Comment:
This seems like much more access than should be granted to bluez, see @jdstrand's comments at https://forum.snapcraft.io/t/best-way-to-access-dev-uinput/16242/5.

Why does bluez need this access? Perhaps the bluez snap should also use the new uinput interface (although it sounds like that interface will be super-privileged, so perhaps bluez shouldn't be using this? See https://github.com/snapcore/snapd/pull/8867

Reply:
> Why does bluez need this access?

One of the use cases is probably is buttons like pause/play on BT headsets/speakers.

Reply:
So yes, this is to let bluez access buttons in BT devices. Anyway, I am fine with dropping this as it is not critical if @jdstrand thinks it is an issue and that we should wait for the uinput interface to exist.

Reply:
We now have a uinput interface. It would be better for the bluez snap to plugs it rather than add it directly here.

Reply:
Removed now.